### DISCUSSION TITLE
print to scale

### DISCUSSION THREAD

**Discussion by hedger48** [OpenOrienteering/mapper#2044] [Questions & Answers]
Hello, i am going a bit crazy.  i run an annual event for scouts.  last year my print of map was good to scale, but i am trying to test print my map on 8.5x11 paper, 1:5000 scale and whether i print it from OOM or open in purple pen and print, it is just a little bit smaller than it should be.  i made sure to check any place where the printer might fit to page or scale, all set to "none" but i wont print accurate to scale.  i.e. my scale on page is 1:5000 and scale for 250m should be 5cm but i always get 4.8cm actual measured.  Has anyone run in to this?  thanks!  (worst case, the scouts wont notice, but i do have some orienteers from the local club that may).

**Comment by Choreuomai** [comment]
Hi, this is probably because of your printer margins. when i print from purple pen, i adjust map scale to the printer - sth like 1:4820, the printer settings can tell you


Am 1. März 2022 21:07:44 MEZ schrieb hedger48 ***@***.***>:
>Hello, i am going a bit crazy.  i run an annual event for scouts.  last year my print of map was good to scale, but i am trying to test print my map on 8.5x11 paper, 1:5000 scale and whether i print it from OOM or open in purple pen and print, it is just a little bit smaller than it should be.  i made sure to check any place where the printer might fit to page or scale, all set to "none" but i wont print accurate to scale.  i.e. my scale on page is 1:5000 and scale for 250m should be 5cm but i always get 4.8cm actual measured.  Has anyone run in to this?  thanks!  (worst case, the scouts wont notice, but i do have some orienteers from the local club that may).
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/OpenOrienteering/mapper/discussions/2044
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by lpechacek** [comment]
Interesting observation. I'm surprised that both Purple Pen and OO Mapper print the map at a different scale. If I attempted to analyze such an issue, I'd try:
1) Exporting the map to PDF. Print with a standard PDF viewer.
2) Selecting A4 paper size in Mapper export/print. Maybe there is a bug in the export routines for different paper sizes. Improbable, but worth checking.
3) Exporting the map from Mapper into .ocd format. Then open the .ocd map in OCAD Viewer and export/print from there.
4) Double-checking in Mapper that the north lines are indeed 50 mm apart. :)
5) Printing the map on a different system (i.e. in a public library, copy centre, friend's computer, ...). I suspect that the problem lies in the printer drivers after all.

Hope that helps. I'll be happy to hear about your findings.

**Reply by hedger48** [reply]
thanks for the repsonse.  I exported to PDF and have the same problem where 5mm prints 4.8mm.  Also tried in OCAD viewer and same thing.  the PDF ruler does show that the expected 5mm measures 5mm, so while i think i am turning off any scaling in the printer setting, i must be missing a setting somewhere.  i'll try printing on a different system next.  thanks again!
